src/tab.rs

```diff
@@ -4411,7 +4451,35 @@ impl Tab {
                     match output.send(message).await {
                         Ok(()) => {}
                         Err(err) => {
-                            log::warn!("failed to send thumbnail for {:?}: {}", path, err);
+                            log::warn!("failed to send thumbnail for {:?}: {}", &path, err);
+                        }
+                    }
+
+                    let mut total_size = 0;
+
+                    if let Ok(entries) = fs::read_dir(path.clone()) {
```

Comment:
Now that you are using walkdir you don't have to do this loop any more

Reply:
Ops, sorry, fixed it